connectivity/netsocket/source/TLSSocketWrapper.cpp

```diff
@@ -427,7 +435,11 @@ void TLSSocketWrapper::print_mbedtls_error(MBED_UNUSED const char *name, MBED_UN
 {
 // Avoid pulling in mbedtls_strerror when trace is not enabled
 #if defined FEA_TRACE_SUPPORT && defined MBEDTLS_ERROR_C
-    char *buf = new char[128];
+    char *buf = new (std::nothrow) char[128];
```

Comment:
This can be a buffer on the stack: `char buf[128]`. We don't need memory allocation for 128 bytes.